<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExecutionPlanDialog</class>
 <widget class="QDialog" name="ExecutionPlanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>43</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="8">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>всего: </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="whatsThis">
        <string>изменить текущую запись</string>
       </property>
       <property name="text">
        <string>Просмотр</string>
       </property>
       <property name="shortcut">
        <string>F4</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrint">
       <property name="text">
        <string>Печать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="whatsThis">
        <string>выйти из списка без выбора</string>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblAPDirectionDate">
     <property name="text">
      <string>Назначено</string>
     </property>
     <property name="buddy">
      <cstring>edtDirectionDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtDirectionDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string>дата начала осмотра</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QTimeEdit" name="edtDirectionTime">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lblAPSetPerson">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Назначил</string>
     </property>
     <property name="buddy">
      <cstring>cmbSetPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="4">
    <widget class="CPersonComboBoxEx" name="cmbSetPerson">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Врач</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblAPBegDate">
     <property name="text">
      <string>Начать с</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string>дата окончания осмотра</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="lblAPEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Закончить</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="lblAPDuration">
     <property name="toolTip">
      <string>Длительность курса лечения в днях.</string>
     </property>
     <property name="text">
      <string>Длительность</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QSpinBox" name="edtDuration">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Длительность курса лечения в днях.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="8">
    <widget class="CExecutionPlanTableView" name="tblExecutionPlan"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CExecutionPlanTableView</class>
   <extends>QTableView</extends>
   <header>ExecutionPlanTableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtDirectionDate</tabstop>
  <tabstop>edtDirectionTime</tabstop>
  <tabstop>cmbSetPerson</tabstop>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>edtDuration</tabstop>
  <tabstop>tblExecutionPlan</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnPrint</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
